### PR TITLE
#221 Fix is avatar is undefined

### PR DESCRIPTION
**Pull request by burnacid**
To help your busy schedule

### CODE REVIEW

File: Upload/jscripts/MentionMe/autocomplete.js [hunk 1]
```diff
@@ -299,7 +299,9 @@ var MentionMe = (function($, m) {
 				avatar = "";
 				if (options.showAvatars) {
 					avatarPath = data[user]["avatar"];
-					if (avatarPath.length == 0) {
+					if (typeof avatarPath == 'undefined'){
+						avatarPath = options.defaultAvatar;
+					}else if (avatarPath.length == 0) {
```
Comment (WildcardSearch) [inline Upload/jscripts/MentionMe/autocomplete.js]: Needs a space after the opening curly brace, please.

File: Upload/jscripts/MentionMe/autocomplete.js [hunk 2]
```diff
@@ -299,7 +299,9 @@ var MentionMe = (function($, m) {
 				avatar = "";
 				if (options.showAvatars) {
 					avatarPath = data[user]["avatar"];
-					if (avatarPath.length == 0) {
+					if (typeof avatarPath == 'undefined'){
```
Comment (WildcardSearch) [inline Upload/jscripts/MentionMe/autocomplete.js]: Please use double quotes `"` around strings in JavaScript.

**Comment by WildcardSearch** [comment]
First of all, thanks for contributing. Please take a look at [CONTRIBUTING.md](https://github.com/WildcardSearch/MentionMe/blob/master/CONTRIBUTING.md) for more info about creating pull requests for this project.

I've done a review and there are two small corrections, but first you will need to rebase your pull request to the `maintenance` branch. Once you solve those simple code changes, I will get this merged and these changes will be in **3.2.5**

**Comment by burnacid** [comment]
Canceled

**Pull request closed without merging**